.eslintignore

```diff
@@ -20,6 +20,10 @@ experimental/babel-preset-env/data
 experimental/babel-preset-env/test/debug-fixtures
 packages/babel-standalone/babel.js
 packages/babel-standalone/babel.min.js
+packages/babylon/build
+packages/babylon/.nyc_output
+packages/babylon/coverage
```

Comment:
We will not have coverage reports in the babylon folder, it works exactly the same as all the babel packages.